js/main.js

```diff
@@ -0,0 +1,27 @@
+var $photoUrlInput = document.querySelector('.photourl');
+var $img = document.querySelector('.imageurl');
+
+function photoInput(event) {
+  if (event.target.matches('.photourl')) {
```

Comment:
This conditional isn't necessary, you applied this event handler directly to this element, so it will never not be that element.